ui/src/Common/TealCheckedCheckboxIcon/TealCheckedCheckboxIcon.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022 Ford Motor Company
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import CheckedCheckboxIcon from 'Assets/CheckedCheckboxIcon';
+import { useRecoilValue } from 'recoil';
+import { ThemeState } from 'State/ThemeState';
+import Theme from 'Types/Theme';
+
+import './TealCheckedCheckboxIcon.scss';
+
+function TealCheckedCheckboxIcon() {
```

Comment:
Why put Teal in the name when it is only Teal if the theme is Dark?

Reply:
It's either teal, or seafoam blue, which is still close to teal, so it conveys the general color of the icon, which is what I wanted to communicate. Do you have a better name suggestion?